index.html

```diff
@@ -33,6 +33,11 @@
 
     <mgt-login></mgt-login>
     <mgt-person person-query="me" view="twoLines" person-card="hover" show-presence></mgt-person>
+    <mgt-person
```

Comment:
We try not to change the index.html file and instead add stories to our storybook - see [wiki](https://github.com/microsoftgraph/microsoft-graph-toolkit/wiki/Storybook)

Could you revert this change and add a story for this instead?

Reply:
I will. Thnx

Reply:
@nmetulev 

Should I create a new story for the fallback-details or can I add it to an existing story like 'Person' or 'More examples'?
If it's a new story, what name should I use?
 
![personStory](https://user-images.githubusercontent.com/36161889/108286962-57f48400-718a-11eb-8601-e1b681d2a6fd.PNG)

Reply:
What do you think about adding it as a new story and naming it `personFallbackDetails`?

Reply:
OK. I will make a new story.

Reply:
I added the new story with 3 examples:

 1. Only display name is provided
 2. Only email is provided
 3. Display name and email are provided

![storyMeganB](https://user-images.githubusercontent.com/36161889/108291027-d7398600-7191-11eb-8507-6bffa7757f0c.PNG)

The parameter `email` has to be `mail`, otherwise the email won't show up.
`getEmailFromGraphEntity` checks only for `user.mail`. Otherwise we need to change the function.

```
export function getEmailFromGraphEntity(entity: IDynamicPerson): string {
  const person = entity as Person;
  const user = entity as User;
  const contact = entity as Contact;

  if (user.mail) {
    return extractEmailAddress(user.mail);
  } else if (person.scoredEmailAddresses && person.scoredEmailAddresses.length) {
    return extractEmailAddress(person.scoredEmailAddresses[0].address);
  } else if (contact.emailAddresses && contact.emailAddresses.length) {
    return extractEmailAddress(contact.emailAddresses[0].address);
  }
  return null;
}
```

Reply:
This is perfect. checking for `mail` is per design as that is what the graph data uses. Thanks for adding the story.

Let me know if you need help with the docs. Once that is added, we can merge this PR